unlock-app/src/components/interface/Header.js

```diff
@@ -5,22 +5,68 @@ import Link from 'next/link'
 import Svg from './svg'
 import Buttons from './buttons/layout'
 
-export default function Header({ forContent, title }) {
-  return (
-    <TopHeader>
-      {!forContent && <Title>{title}</Title>}
-      {!!forContent && (
-        <Link href="/">
-          <a>
-            <Svg.UnlockWordMark height="28px" width="100%" name="Unlock" />
-          </a>
-        </Link>
-      )}
-      <Buttons.About />
-      <Buttons.Jobs />
-      <Buttons.Github />
-    </TopHeader>
-  )
+export default class Header extends React.PureComponent {
+  constructor(props) {
+    super(props)
+    this.forContent = props.forContent
+    this.title = props.title
+    this.toggleMenu = this.toggleMenu.bind(this)
+    this.state = { menu: false }
+  }
+
+  toggleMenu() {
+    this.setState(prevState => ({ menu: !prevState.menu }))
+  }
+
+  render() {
+    const { menu } = this.state
+
+    return (
+      <TopHeader>
+        {!this.forContent && <Title>{this.title}</Title>}
+        {!!this.forContent && (
+          <Link href="/">
+            <a>
+              <Svg.UnlockWordMark height="28px" width="100%" name="Unlock" />
+            </a>
+          </Link>
+        )}
+        <DesktopButtons>
+          <Buttons.About />
+          <Buttons.Jobs />
+          <Buttons.Github />
+        </DesktopButtons>
+        <MobileToggle>
+          <Buttons.Bars
+            onClick={this.toggleMenu}
+            size="48px"
+            className={
+              'mobile-toggle ' +
+              (menu ? 'mobile-toggle-invisible' : 'mobile-toggle-visible')
+            }
+          />
+          <Buttons.ChevronUp
+            onClick={this.toggleMenu}
+            size="48px"
+            className={
+              'mobile-toggle ' +
+              (menu ? 'mobile-toggle-visible' : 'mobile-toggle-invisible')
+            }
+          />
+        </MobileToggle>
+        <MobilePopover
+          className={menu ? 'popover-visible' : 'popover-invisible'}
+        >
+          <Buttons.About size="48px" className="mobile-menu" />
+          <Buttons.Jobs size="48px" className="mobile-menu" />
+          <Buttons.Github size="48px" className="mobile-menu" />
+          <Buttons.Newsletter size="48px" className="mobile-menu" />
```

Comment:
You can remove the Newsletter, Telegram and Twitter buttons because we actually do not use them yet... (I know the design brief is wrong on that... sorry!)

Reply:
No worries! They can easily be popped in later. It's nice to have them available anyway.